go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	cosmossdk.io/errors v1.0.1
 	cosmossdk.io/math v1.3.0
 	github.com/celestiaorg/blobstream-contracts/v3 v3.1.0
+	github.com/celestiaorg/go-square v1.0.0
 	github.com/celestiaorg/go-square/v2 v2.0.0-rc2
```

Comment:
[separate PR] should we create an official release of go-square so that it can be used here?

Reply:
Yes, I noticed this as well

Reply:
Everything seems to be passing so it's probably safe to cut the final release